Source/VMware.vSphereDSC/Classes/InventoryBaseDSC.ps1

```diff
@@ -18,83 +18,188 @@ class InventoryBaseDSC : BaseDSC {
     <#
     .DESCRIPTION
 
+    Name of the Inventory Item from the Datacenter we will use.
+    #>
+    [DscProperty(Key)]
+    [string] $Name
+
+    <#
+    .DESCRIPTION
+
+    The full path to the Inventory Item in the Datacenter we will use from the Inventory.
```

Comment:
Inventory folder path location of the resource with name specified on Name key property in the Datacenter specified in the 'Datacenter' key property